### ISSUE TITLE
[🐛 Bug]: SM does not pull latest minor releases for previous versions of firefox

### ISSUE THREAD

**Issue by elliottj** [SeleniumHQ/selenium#14536]
### What happened?

Running Selenium Manager Windows 0.4.25 command line to pull latest Firefox 127 version shows:
```
c:\Temp>selenium-manager-windows.exe --browser firefox --browser-version 127 --debug
[2024-09-25T11:47:46.705Z DEBUG] Sending stats to Plausible: Props { browser: "firefox", browser_version: "127", os: "windows", arch: "amd64", lang: "", selenium_version: "4.25" }
[2024-09-25T11:47:46.749Z DEBUG] geckodriver not found in PATH
[2024-09-25T11:47:46.749Z DEBUG] firefox detected at C:\Program Files\Mozilla Firefox\firefox.exe
[2024-09-25T11:47:46.750Z DEBUG] Running command: wmic datafile where name='C:\\Program Files\\Mozilla Firefox\\firefox.exe' get Version /value
[2024-09-25T11:47:46.981Z DEBUG] Output: "\r\r\n\r\r\nVersion=130.0.1.967\r\r\n\r\r\n\r\r\n\r"
[2024-09-25T11:47:46.985Z DEBUG] Detected browser: firefox 130.0.1.967
[2024-09-25T11:47:46.987Z DEBUG] Discovered firefox version (130) different to specified browser version (127)
[2024-09-25T11:47:46.990Z DEBUG] Required browser: firefox 127.0
[2024-09-25T11:47:46.990Z DEBUG] Downloading firefox 127.0 from https://ftp.mozilla.org/pub/firefox/releases/127.0/win64/en-US/Firefox%20Setup%20127.0.exe
```
It's downloading **127.0** but, as per https://www.mozilla.org/en-US/firefox/releases/, the current latest version is **127.0.2**.

First noticed when using Selenium 4.25 release, previously working under Selenium 4.18.

Likely linked to 0.42 change:
"Use Firefox history major releases endpoint for version discovery"

### How can we reproduce the issue?

```shell
See above.
```

### Relevant log output

```shell
See above.
```

### Operating System

Windows 10

### Selenium version

Java 4.24.0

### What are the browser(s) and version(s) where you see this issue?

Firefox

### What are the browser driver(s) and version(s) where you see this issue?

Any version that is not latest, but has it's own later minor version.

### Are you using Selenium Grid?

n/a

**Comment by elliottj** [comment]
Any updates on this please?

**Comment by diemol** [comment]
@bonigarcia do you know?

**Comment by bonigarcia** [comment]
Yes, I have confirmed this. Many thanks for reporting, @elliottj. I will do a fix soon.

**Comment by elliottj** [comment]
Is there an update on this please, it looks like the fix failed to build? Thanks.

**Comment by diemol** [comment]
4.26 has been released, can you check with that version?

**Comment by bonigarcia** [comment]
I believe that PR was not merged, so I'm afraid release 4.26 will work in the same way:

https://github.com/SeleniumHQ/selenium/pull/14613

**Comment by elliottj** [comment]
Is there an estimate on when the PR will be fixed and merged please?

**Comment by bonigarcia** [comment]
The PR that fixes this issue has been merged. It will be available in the next Selenium release.

**Comment by elliottj** [comment]
Thanks is fixed in released 4.27.0.
